app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+  end
+
+  def create
+    @movie = Movie.find(params[:id])
+    @actor = Actor.find_by(id: params[:actor_id])
+    @movie.actors << @actor
+    redirect_to "/movies/#{@movie.id}"
+  end
```

Comment:
I agree with your submission comment that we probably want to create this movie actor relationship in the `MovieActor` controller. I would expect the `create` action of the `MoviesController` to create a movie, not to add an existing actor to an existing movie.